portal/views/user.py

```diff
@@ -1604,3 +1604,63 @@ def posted_filename(request):
         doc.uuid, user_id), user_id=current_user().id, subject_id=user_id,
         context='assessment')
     return jsonify(message="ok")
+
+@user_api.route('/user/<int:user_id>/password_reset', methods=('POST',))
+@oauth.require_oauth()
+def trigger_password_reset_email(user_id):
+    """Trigger a password reset email for the specified user
+
+    Allows admins, staff, etc to manually trigger password reset emails to
+    patients, allowing them to easily change their passwords and login without
+    knowing their current password or email.
+    ---
+    tags:
+      - User
+      - Password
```

Comment:
These tags build additional categories in the swagger docs.  I think we should avoid lots of tags until the tree is deep and unruly.  For now, lets just use 'User' for this.